pkg/asset/machines/machineconfig/manifest.go

```diff
@@ -0,0 +1,67 @@
+package machineconfig
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+
+	"github.com/openshift/installer/pkg/asset"
+)
+
+const (
+	machineConfigFileName = "99_openshift-machineconfig_%s.yaml"
+)
+
+var (
+	machineConfigFileNamePattern = fmt.Sprintf(machineConfigFileName, "*")
+)
+
+// Manifests creates manifest files containing the MachineConfigs.
+func Manifests(configs []*mcfgv1.MachineConfig, role, directory string) ([]*asset.File, error) {
+	data := []byte{}
+	for _, c := range configs {
+		if c == nil {
+			continue
+		}
+		configData, err := yaml.Marshal(c)
+		if err != nil {
+			return nil, err
+		}
+		data = append(data, []byte("---\n")...)
```

Comment:
Ugh :p.  Can we just `append` a separate `*asset.File` for each MachineConfig?

Reply:
keeping different files doesn't get us a lot! ;) keeping as-is.